src/components/ActionBars/OperatorActionBar.tsx

```diff
@@ -199,11 +189,19 @@ export const OperatorActionBar: FunctionComponent<{
                     <NetworkActionBarCTAs>
                         <Button
                             onClick={async () => {
+                                if (!walletAddress) {
+                                    return
+                                }
+
                                 try {
-                                    await delegateFunds(operator)
+                                    await delegateFunds({
```

Comment:
There's no utility currently in having `delegateFunds` and `undelegateFunds` as something that comes from a react hook, after all. I moved it out.

There's a decent chance they'll come back wrapped with something that actually makes sense, for instance, something tracks progress.